src/read/elf/relocation.rs

```diff
@@ -958,4 +964,8 @@ impl<'data> Iterator for CrelIterator<'data> {
         }
         Some(result)
     }
+
+    fn size_hint(&self) -> (usize, Option<usize>) {
+        (self.len(), Some(self.len()))
```

Comment:
Technically I think this should be `(max(1, self.len()), Some(self.len())`, since an error in parsing could mean we only return one more item. I'm not familiar with what this is used for to know if this matters.